functional_tests/functional_test.go

```diff
@@ -167,7 +167,7 @@ var _ = ginkgo.Describe("Tests `appgw.ConfigBuilder`", func() {
 				annotations.IngressClassKey: annotations.ApplicationGatewayIngressClass,
 			},
 			Namespace: tests.Namespace,
-			Name:      tests.Name,
+			Name:      tests.Name + "FooBazNoTLS",
```

Comment:
Is there any specific reason to add suffix here?

Reply:
yes, some test use multiple ingresses and it would be not be right to have the same names for them.